rotkehlchen/tests/unit/test_general_cache.py

```diff
@@ -110,8 +147,31 @@ def mock_call_contract(contract, node_inquirer, method_name, **kwargs):
         new=mock_call_contract,
     )
 
+    requests_get = requests.get
+
+    def mock_requests_get(url, timeout):  # pylint: disable=unused-argument
```

Comment:
you need a docstring here to explain what you mock and why, especially considering this is a vcred test